packages/test/test-end-to-end-tests/src/test/audience.spec.ts

```diff
@@ -10,12 +10,12 @@ import {
 	timeoutPromise,
 	waitForContainerConnection,
 } from "@fluidframework/test-utils";
-import { describeFullCompat } from "@fluid-internal/test-version-utils";
+import { describeFullVersionCompat } from "@fluid-internal/test-version-utils";
```

Comment:
Do we still distinguish the cases where we would run describeFullCompat vs describeFullVersionCompat? I was hopeful that top-level configuration (specificed by pipeline for example), can decide compat matrix it wants to run.